src/contexts/McAppContext.tsx

```diff
@@ -49,6 +58,28 @@ export const McAppContextProvider: React.FC<McAppContextProviderProps> = ({
         AsyncStatus.LOADING
     );
 
+    const [offlineState, setOfflineState] = useState(false);
+    const statusRepository = new StatusRepository();
+
+    useEffect(() => {
+        const asyncFunction = async (): Promise<void> => {
+            const status = await statusRepository.getStatus();
+            setOfflineState(
+                status.status
```

Comment:
Dette ser ikke ut til å fungere hos meg. Når offlineStorage i indexedDB er tom så får jeg error + ingenting kommer i indexedDb. Prøv å kun sjekke status i stedet for status.status på linje 68 f.eks.